### PR TITLE
ISPN-7503 Resolve expressions in the endpoint subsystem

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-7503

### CODE REVIEW

File: server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/MemcachedSubsystemAdd.java [hunk 1]
```diff
@@ -58,12 +58,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
       MemcachedServerConfigurationBuilder configurationBuilder = new MemcachedServerConfigurationBuilder();
       this.configureProtocolServer(configurationBuilder, config);
 
-      final String cacheName;
-      if (config.hasDefined(ModelKeys.CACHE)) {
-         cacheName = config.get(ModelKeys.CACHE).asString();
-      } else {
-         cacheName = MemcachedServerConfiguration.DEFAULT_MEMCACHED_CACHE;
```
Comment (ryanemerson) [inline server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/MemcachedSubsystemAdd.java]: Unused import `MemcachedServerConfiguration`

File: server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/RouterConnectorResource.java [hunk 2]
```diff
@@ -18,6 +18,8 @@
  */
 package org.infinispan.server.endpoint.subsystem;
 
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
+import org.infinispan.server.hotrod.configuration.HotRodServerConfiguration;
```
Comment (ryanemerson) [inline server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/RouterConnectorResource.java]: Unused imports: 

```
import org.infinispan.client.hotrod.impl.ConfigurationProperties;
import org.infinispan.server.hotrod.configuration.HotRodServerConfiguration;
```

**Comment by tristantarrant** [comment]
@ryanemerson I have removed the unused imports.

**Comment by pruivo** [comment]
look good. pushing...

**Comment by pruivo** [comment]
integrated! thanks @tristantarrant

**Pull request merged**